pkg/storegateway/bucket_test.go

```diff
@@ -2626,7 +2628,7 @@ func benchmarkBlockSeriesWithConcurrency(b *testing.B, concurrency int, blockMet
 					MinTime:    blockMeta.MinTime,
 					MaxTime:    blockMeta.MaxTime,
 					Matchers:   reqMatchers,
-					SkipChunks: false,
```

Comment:
I removed the support to fetch chunks from `blockSeriesSkippingChunks()` (reason why function was renamed too) so doesn't matter the value of `SkipChunks` anymore because the function will never fetch chunks.